triage/triaging.rst

```diff
@@ -62,44 +53,45 @@ For bugs, an issue needs to:
 These are things you can help with once you have experience developing for
 Python:
 
-* try reproducing the bug: For instance, if a bug is not clearly explained
-  enough for you to reproduce it then there is a good chance a core developer
+* Try reproducing the bug: for instance, if a bug is not explained clearly
```

Comment:
```suggestion
* Try reproducing the bug: if it is not explained clearly
```

The "for instance" is redundant and the phrasing sounds off, better to just proceed directly to the point, IMO